extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -420,7 +420,7 @@
 # formats 'table' and 'list' are defaults that cannot be disabled. The format
 # 'broadtable' should not be disabled either in order not to break Special:ask.
 ##
-$GLOBALS['smwgResultFormats'] = array(
+$smwDefaults['smwgResultFormats'] = array(
 	'table'      => 'SMW\TableResultPrinter',
 	'list'       => 'SMW\ListResultPrinter',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SMW\ListResultPrinter" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)